xunit-autolabeler-v2/ast_parser/python/source_parsers/test_data/flask_router_example.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from flask import Flask
+from werkzeug.routing import Rule
+
+
+app = Flask(__name__)
+
+# This is a valid Flask decorator, but it isn't used in
+# our samples and is thus ignored by the flask_router parser
```

Comment:
@tmatsuo I couldn't find this pattern when I looked, but I'll let you LGTM this.

Reply:
I'm not sure about this comment. The fact that there is no examples doesn't ensure we won't have it in the future.

Reply:
If we _do_ add support for this in the future, it will cause a test failure (which, at least theoretically, would flag the discrepancy).

Happy to add a clarifying comment here along these lines:
```
(The tests depend on this behavior, and will need to be modified in the event support *is* added in the future.)
```

Reply:
Hm, my point is, we will potentially have samples with this style in the future.
If that happens, this tool can not handle such samples.

Am I wrong?

If I'm right, I'd like us to have support for it from the start.

Reply:
Strictly speaking, _potential_ future sample formats are largely unknowable - and thus, the _number_ of them is essentially infinite. Supporting all (or even most) of a near-infinite format set with a deterministic parser like this is essentially impossible. (You would likely need some sort of AI/ML approach to truly support arbitrary formats.)

If we're _actively planning_ to add _this particular feature_ to our samples, I'm happy to support it. Otherwise - I'm worried this will be a (literally infinitely) deep hole to go down - and it's probably better covered via tests than an explicit implementation.

Reply:
Well, I'm not asking you to add infinite stuff. There are only handful of patterns for flask routes, and this is one of them right?

However, I can LGTM without implementing it.